### ISSUE TITLE
Doesn't boot to gsi

### ISSUE THREAD

**Issue by BluBerryPlayz** [VegaBobo/DSU-Sideloader#151]
Everything installed perfectly fine in my Realme Narzo 20A (rmx2050) but when I reboot using the dsu notification it reboots to system instead of the gsi

**Comment by ghost** [comment]
Did you experience any attempted boots into the gsi (boot logo loading multiple times)? It is normal that not all GSIs boot. Have you tried other GSIs?

**Comment by BluBerryPlayz** [comment]
> Did you experience any attempted boots into the gsi (boot logo loading multiple times)? It is normal that not all GSIs boot. Have you tried other GSIs?

Yes I saw it attempted to boot, then black screen, and then it reboots to stock os. I have tried more than 1 gsi, pixel os gsi, pixel experience gsi and nippon's one UI 5 gsi

**Comment by carloanselmini** [comment]
Happens to me too, pretty much like OP, however only when I use zips containing more than one image, for example a system.img and a product.img, but i f I try the single system.img it works
